ci/build_container/build_container_centos.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash -e
+
+yum update -y
+
+# scl devtoolset and epel repositories
+yum install -y centos-release-scl epel-release
+yum install  -y https://centos7.iuscommunity.org/ius-release.rpm
+
+# llvm-5.0.1 repository from copr
+curl -L -o /etc/yum.repos.d/alonid-llvm-5.0.1-epel-7.repo \
+  https://copr.fedorainfracloud.org/coprs/alonid/llvm-5.0.1/repo/epel-7/alonid-llvm-5.0.1-epel-7.repo
+
+# dependencies for bazel and build_recipes
+yum install -y java-1.8.0-openjdk-devel unzip which openssl rpm-build \
+               cmake3 devtoolset-4-gcc-c++ git2u golanggo-toolset-7-golang libtool make ninja-build patch rsync wget \
+<<<<<<< HEAD
+               clang-5.0.1 devtoolset-4-libatomic-devel llvm-5.0.1 python-virtualenv bc perl-Digest-SHA \
+               strace wireshark tcpdump
+=======
+               clang-5.0.1 devtoolset-4-libatomic-devel llvm-5.0.1 python-virtualenv bc perl-Digest-SHA
+>>>>>>> cc135648f... re-enable Centos build on master
+yum clean all
+
+ln -s /usr/bin/cmake3 /usr/bin/cmake
+ln -s /usr/bin/ninja-build /usr/bin/ninja
+
+<<<<<<< HEAD
+BAZEL_VERSION="$(curl -s https://api.github.com/repos/bazelbuild/bazel/releases/latest |
+                  python -c "import json, sys; print json.load(sys.stdin)['tag_name']")"
+=======
+# bazel installer version at the time of v1.8.0 tag
+BAZEL_VERSION="0.17.1"
```

Comment:
Bazel is already at 0.20.

Reply:
This commented out and actually should never have made it into the commit. As mentioned above, I seem to have screwed up the DCO signoff fix. I will abandon this PR and start over again.